projects/sonar/src/app/_layout/admin/admin.component.spec.ts

```diff
@@ -16,12 +16,16 @@
  */
 import { HttpClientModule } from '@angular/common/http';
```

Comment:
imports can be sorted

Reply:
They're sorted, why do you think they're not?